### ISSUE TITLE
Birds of a Feather for RustConf 2017

### ISSUE THREAD

**Issue by ashleygwilliams** [rust-community/team#181]
this is a brainstorming and tracking issue for running a BOF at RustConf.

@aturon said he'd give this a shot.

my number estimate:
> i would estimate that we should expect ~10-20% of people to participate, which is 25-50 people, 5 topics = 5-10 people per table

so if we can get 5 topics going that'd be great. thanks ya'll :)

**Comment by aturon** [comment]
I talked to Leah about this today, and we weren't able to find a solution that seemed solid logistically. People tend to sit at the same table throughout the day, and asking folks to shuffle during lunch seemed chaotic. I suggested we could have signage from the beginning of the day, but she felt it'd be largely ignored. And unfortunately, we're tight enough in space in the venue that a hallway BOF doesn't seem too workable either.

Maybe there's a way we could coordinate this out of band? E.g. set up dinner groups with like-minded folks?

Anybody have other ideas?
